code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -513,16 +513,16 @@
 		var/treatment_method
 		var/list/potential_methods = list()
 
-		if(C.getBruteLoss() >= heal_threshold)
+		if((C.getBruteLoss() >= heal_threshold) && C.getBruteLoss() != 0)
 			potential_methods += BRUTE
 
-		else if(C.getFireLoss() >= heal_threshold)
+		else if((C.getFireLoss() >= heal_threshold) && C.getFireLoss() != 0)
```

Comment:
Should these be `else` ?

Reply:
No, the `else`s aren't really needed, I'll get rid of them.

Reply:
One thing I noticed with removing these `else`s, where the `damagetype_healer = "all"` bots would previous always heal in the order listed (so brute, then burn, oxy etc etc), they now basically pick a random type to heal each tick. Not a big deal, I think, but something worth considering.